arctic/_util.py

```diff
@@ -23,7 +23,7 @@ def are_equals(o1, o2, **kwargs):
         return False
 
 
-def enable_sharding(arctic, library_name, hashed=True):
+def enable_sharding(arctic, library_name, hashed=True, key='symbol'):
```

Comment:
I'm happy to change to a list (keys), or we can add support for it later if/when it becomes useful.